modules/nw-ovn-kubernetes-matrix.adoc

```diff
@@ -47,7 +51,11 @@ ifeval::["{context}" == "about-openshift-sdn"]
 
 |IPsec encryption|Not supported|Supported
 
-|IPv6|Not supported|Supported ^[3]^ ^[4]^
+|IPv4-primary dual-stack|Supported|Supported ^[3]^
+
+|IPv6 single-stack|Supported ^[4]^|Unsupported
+
+|IPv6-primary dual-stack|Supported ^[5]^|Unsupported
```

Comment:
Ah, It looks like the columns were not swapped correctly for OpenShift SDN.

should be 

```
|IPv4-primary dual-stack|Unsupported|Supported ^[3]^

|IPv6 single-stack|Unsupported|Supported ^[4]^

|IPv6-primary dual-stack|Unsupported|Supported  ^[5]^
```